execution/src/windows/execution.cc

```diff
@@ -313,6 +313,11 @@ namespace leatherman { namespace execution {
                     if (pipe.read) {
                         // Read completed immediately, process the data
                         pipe.buffer.resize(count);
+                        if (convert_newlines) {
+                            pipe.buffer.erase(
+                                std::remove(pipe.buffer.begin(), pipe.buffer.end(), '\r'),
```

Comment:
Are there ever times you'd legitimately want the CR (when clearing end-of-line CR)? I can't think of any.